File: main/src/main/java/net/fwitz/math/main/binary/dual/functions/trig/DualCosZ.java
package net.fwitz.math.main.binary.dual.functions.trig;

import net.fwitz.math.binary.dual.DualNumber;
import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot;

public class DualCosZ {
    public static void main(String[] args) {
        new DualNumberFunctionPlot("cos(z)")
                .computeFn(DualNumber::cos)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/dual/functions/trig/DualCosZ.kt
package net.fwitz.math.main.binary.dual.functions.trig

import net.fwitz.math.plot.binary.dual.DualNumberFunctionPlot

object DualCosZ {
    @JvmStatic
    fun main(args: Array<String>) = DualNumberFunctionPlot("cos(z)")
        .computeFn { z -> z.cos }
        .render()
}